src/Microsoft.AspNet.Mvc.ViewFeatures/Controller.cs

```diff
@@ -424,6 +424,33 @@ public virtual JsonResult Json(object data, [NotNull] JsonSerializerSettings ser
         }
 
         /// <summary>
+        /// Creates a <see cref="HttpOkResult"/> object that produces an empty OK (200) response.
+        /// </summary>
+        /// <returns>The created <see cref="HttpOkResult"/> for the response.</returns>
+        [NonAction]
+        public virtual HttpOkResult Ok()
```

Comment:
Others? You mean like `Created()`, `Redirect()` and `RedirectPermanent()`? I agree we've also got `HttpBadRequest()`, `HttpNotFound()` and `HttpUnauthorized()` but why is that method naming convention preferred?